app/src/main/res/values-v29/themes.xml

```diff
@@ -24,6 +24,7 @@
         <item name="android:navigationBarColor">@android:color/transparent</item>
         <item name="android:statusBarColor">@android:color/transparent</item>
         <item name="android:windowLightStatusBar">?attr/isLightTheme</item>
+        <item name="android:forceDarkAllowed" tools:targetApi="q">false</item>
```

Comment:
Please revert this change. We only have to control the forceDarkAllowed attribute in the parent theme.